src/Propel/Generator/Builder/Om/ObjectBuilder.php

```diff
@@ -5834,11 +5837,13 @@ public function set{$relatedNamePlural}(Collection \${$inputCollection}, Connect
     /**
      * @param string $script
      * @param \Propel\Generator\Model\CrossForeignKeys $crossFKs
+     *
+     * @return void
      */
     protected function addCrossFKCount(&$script, CrossForeignKeys $crossFKs)
     {
         $refFK = $crossFKs->getIncomingForeignKey();
-        $selfRelationName = $this->getFKPhpNameAffix($refFK, $plural = false);
```

Comment:
We have this issue in 44 places in this file currently

Reply:
I fixed those now, too.